test/bench/report_generator.py

```diff
@@ -169,6 +169,19 @@ def makeSummaryGraph(outputDirectory, summary):
 def splitTagSemVer(tag):
     return list(filter(None, semVerRegex.split(tag)))
 
+# if the string is a number, return it. Otherwise convert
+# the string to a numeric value that sorts alphabetically
+# Assumes characters have a max value of 255.
+def numericValueOf(s):
+    try:
+        return int(s)
+    except ValueError:
+        accumulate = 0
+        ts = s[:min(7, len(s))]
+        for ndx in reversed(range(0, len(ts))):
```

Comment:
I think `for (ndx, chr) in reversed(enumerated(ts)):` would be more ideomatic

Reply:
I agree, I guess I've been C++ ized for too long. Updated.